src/praisonai/praisonai/cli/commands/n8n.py

```
@@ -0,0 +1,321 @@
+                workflow_id = workflow.get('id', 'Unknown')
+                name = workflow.get('name', 'Untitled')
+                active = workflow.get('active', False)
+                status = "✅ Active" if active else "⏸️  Inactive"
+                
+                typer.echo(f"  {workflow_id}: {name} ({status})")
+        
+        client.close()
+        
+    except ImportError:
+        typer.echo("Error: n8n dependencies not installed. Run: pip install 'praisonai[n8n]'", err=True)
+        raise typer.Exit(1)
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)